cpp_build/Cargo.toml

```diff
@@ -18,8 +18,8 @@ docs-only = []
 
 [dependencies]
 lazy_static = "1.0"
-cc = "1.0.38"
-cpp_common = { path = "../cpp_common", version = "=0.5.7" }
+cc = { version = "1.0", features = ["parallel"] }
```

Comment:
why did you remove the .38?   this crate won't work with older version of `cc`

Reply:
I was unaware it does not work with older versions of cc. I'm currently using my branch and have not seen any regression. Could you explain the .38 pin?

Reply:
this is not a pin.  With cargo, this is equivalent to "^1.0.38" which means later version are also accepted.
It needs the .38 because it uses some API that is only available in that version: (eg as explained in https://github.com/mystor/rust-cpp/commit/40f33243219fb7d91d65bf66c124b3eef3bf1042 )
So this is avoiding the (admittedly unlikely at this point) case if someone had a very old Cargo.lock with a very old version of cc, and adds the cpp-build crate.